workers/social/lib/social/models/account.coffee

```diff
@@ -367,6 +373,38 @@ module.exports = class JAccount extends jraphical.Module
           callback null, groups
 
 
+  fetchInviteGroups$: secure (client, options, callback) ->
+
+    @fetchInviteGroups options, callback
+
+
+  fetchInviteGroups: (options, callback) ->
+
+    [ options, callback ] = [ callback, options ]  unless callback
+    options ?= {}
+
+    queue = [
+      (next) =>
+        @fetchEmail next
+      (email, next) ->
+        JInvitation = require './invitation'
+        JInvitation.some { email, status : 'pending' }, options, next
+      (invitations, next) ->
+        { uniq, map } = _
+        slugs  = uniq map invitations, (invitation) -> invitation.groupName
+        JGroup = require './group'
+        JGroup.some { slug : { $in : slugs } }, {}, (err, groups) ->
```

Comment:
I added `limit` to `JInvitation.some()` call above. I thought about usage of `someData()` here but since result of this method is combined with `fetchAllParticipatedGroups()` result we need to return a list of groups to be consistent